Formatting the changelog in preparation for v2.21.0 (#6353)
pulumi/pulumi: CHANGELOG.md

## HEAD (Unreleased)

> ### Improvements

Reviewer: We're missing the 2.21.0 header

> - [cli] Adding pagination options to `pulumi stack history` [#6292](https://github.com/pulumi/pulumi/pull/6292)  

Reviewer: Let's stick w/ past tense.
```suggestion
- [cli] Added pagination options to `pulumi stack history` [#6292](https://github.com/pulumi/pulumi/pull/6292)  
```

  This is used as follows:  
  `pulumi stack history --page-size=20 --page=1`
  

> - [automation/*] Adding pagination options for stack history in Automation API SDKs to improve 

Reviewer: ```suggestion
- [automation/*] Added pagination options for stack history in Automation API SDKs to improve 
```

  performance of stack updates. [#6257](https://github.com/pulumi/pulumi/pull/6257)    
  This is used similar to the following example in go:  
```go
  func ExampleStack_History() {
	ctx := context.Background()
	stackName := FullyQualifiedStackName("org", "project", "stack")
	stack, _ := SelectStackLocalSource(ctx, stackName, filepath.Join(".", "program"))
	pageSize := 0
	page := 0
	hist, _ := stack.History(ctx, pageSize, page)
	fmt.Println(hist[0].StartTime)
  }
```
  

> - [pkg/testing/integration] Change the default behavior for Python test projects to use `UseAutomaticVirtualEnv` by

Reviewer: ```suggestion
- [pkg/testing/integration] Changed the default behavior for Python test projects to use `UseAutomaticVirtualEnv` by
```

  default. `UsePipenv` is now the way to use pipenv with tests.
  [#6318](https://github.com/pulumi/pulumi/pull/6318)

### Bug Fixes

> - [automation/go] Exposing the version in the UpdateSummary for use in understanding the version of a stack update

Reviewer: ```suggestion
- [automation/go] Exposed the version in the UpdateSummary for use in understanding the version of a stack update
```

  [#6339](https://github.com/pulumi/pulumi/pull/6339)

> - [cli] Look for `python` binary first on Windows instead of `python3`.

Reviewer: ```suggestion
- [cli] Changed the behavior for Python on Windows to look for `python` binary first instead of `python3`.
```

  [#6317](https://github.com/pulumi/pulumi/pull/6317)
  
- [sdk/python] Gracefully handle monitor shutdown in the python runtime without exiting the process. 
  [#6249](https://github.com/pulumi/pulumi/pull/6249)

> - [sdk/python] Fix a bug in `contains_unknowns` where outputs with a property named "values" fails with a TypeError.

Reviewer: ```suggestion
- [sdk/python] Fixed a bug in `contains_unknowns` where outputs with a property named "values" failed with a TypeError.
```
